### PR TITLE
Spencer/eng 530 forcing infinite login requests

### PR DESCRIPTION
**Pull request by spencerjsmall**
approach is to use chrome.local.storage instead of web local or session storage

### CODE REVIEW

**Comment by spencerjsmall** [comment]
@joelrojo Re: the last two commits. I think chrome.storage.local is gonna be our solution but for some reason the token isn't being stored properly in agent. The extension loads with the user logged in, but further API requests (get knots, primers) are forbidden. 
Screen Shot 2020-08-06 at 12 18 13 PM

**Comment by spencerjsmall** [comment]
The token persists in storage after the extension is closed/reloaded tho so this should work eventually

**Comment by joelrojo** [comment]
Hmm I would assume it has to do with the agent not setting the token correctly.

```
agent.setToken()
```
Look for how we set that and make sure it's begin set correctly.

Also, noticed this has the same commit history as PR #34 so going to close that in favor of this to keep things clean

File: src/contents/all/components/app/App.tsx [hunk 1]
```diff
@@ -115,14 +115,20 @@ const App = () => {
     }
 
     if (request.message === "clicked_browser_action") {
-      if (user) {
-        setTimeout(() => {
+      chrome.storage.local.get("analogueJWT", function(token) {
+        if (!user && Object.keys(token).length !== 0) {
+          agent.setToken(token.analogueJWT)
+          //dispatch({ type: 'SET_USER', user: **GET USER FROM TOKEN** })
```
Comment (spencerjsmall) [inline src/contents/all/components/app/App.tsx:121]: @joelrojo This is the last missing piece. Need a way to get and dispatch User from a stored token. was thinking agent.Auth.current() but i don't think that does it. Any ideas?

**Pull request closed without merging**